boards.json

```diff
@@ -0,0 +1,9 @@
+{
+    "createNewBoard": "[id='new-board']",
+    "createNewBoardInput": "[id='new-board'] > input",
+    "saveNewBoardButton": "[class='board_options'] > button",
+    "cancelCreateNewBoard" : "[class='board_options'] > svg",
+    "optionsBoardDropdown": "[data-cy='optionsDropdown']",
+    "copyBoardDropdown": "[data-cy='copyDropdown']",
+    "deleteBoardDropdown": "[data-cy='deleteBoardDropdown']"
```

Comment:
Please do not use `data-cy` here, although this is the best possible option - try using whichever other selector you have available. 

The goal of this is to not get too comfortable with high quality selectors, as sometimes you will not have the luxury of having them, nor will you be able to add them yourself, and you will need to use whatever you have.